### PR TITLE
Quantum's Full Voice "Upgrade"

### PR DESCRIPTION
**Pull request by Kashalls**
### Description of Your Pull Request
What does your pull request specfically does by adding/modifying or removing code?

Just see https://github.com/AdityaTD/PenguBot/pull/23

https://uwu.whats-th.is/aa453f.png

### CODE REVIEW

**Comment by Kashalls** [comment]
Seems like Codacy doesn't like Quantums branch and crashes.

**Comment by MrJacz** [comment]
Codacy was meant to be disabled CC @AdityaTD

File: commands/Music/queue.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ module.exports = class extends Command {
         super(...args, {
             runIn: ["text"],
             cooldown: 10,
-            aliases: ["stopmusic", "leave"],
```
Comment (MrJacz) [inline commands/Music/queue.js]: Why were these aliases removed?

File: lib/structures/PenguClient.js [hunk 2]
```diff
@@ -34,6 +22,18 @@ const pLevels = new PermissionLevels()
     // Allows the bot owner to use Bot Owner only commands, which silently fail for other users.
     .add(10, (c, m) => m.author === c.owner);
 
+class PenguClient extends Client {
+
+    constructor() {
```
Comment (MrJacz) [inline lib/structures/PenguClient.js]: you need to pass options

File: lib/structures/PenguClient.js [hunk 3]
```diff
@@ -34,6 +22,18 @@ const pLevels = new PermissionLevels()
...
+        super({ permissionLevels });
```
Comment (MrJacz) [inline lib/structures/PenguClient.js]: this should be `{ ...options, permissionLevels }`

File: commands/Music/queue.js (hunk 1 above)
Comment (Kashalls) [reply commands/Music/queue.js]: You want to trigger stopmusic and leave commands on the queue command? The user shouldn't stop playing music when they fetch the queue.

Comment (MrJacz) [reply commands/Music/queue.js]: Lmfao, you're not wrong. mb

**Pull request merged**